network-api/networkapi/fellows/urls.py

```diff
@@ -3,11 +3,26 @@
 
 urlpatterns = [
     url(r'^$', views.fellows_home, name='fellowships-home'),
+    url(r'^science/$', views.fellows_science, name='fellowships-science'),
+    url(r'^open-web/$', views.fellows_openweb, name='fellowships-open-web'),
     url(r'^directory/$',
         views.fellows_directory,
         name='fellowships-directory'),
+    url(r'^directory/senior$',
```

Comment:
I know we're doing things pretty statically, but I think it's worth it to make this DRYer. Check out [named groups](https://djangobook.com/advanced-views-urlconfs/#leanpub-auto-named-groups)